Resources/deployments/function-app-resources.json

```diff
@@ -0,0 +1,129 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
Is there any reason we don't use the templates from dfc-devops here?
I'm assuming it's so we keep everything working in a similar manner